app/views/entity/list.html.php

```diff
@@ -1,7 +1,11 @@
 <h2>Lijst van Interesse's</h2>
 
 <ul>
-<?php foreach ($list as $item) : ?>
-   <li><?php echo $item->name ?></li>
-<?php endforeach ?>
+    <?php if (count($list) == 0) : ?>
```

Comment:
empty mag ook :+1: 
count === 0 is nog beter :+1: 

de if-else, die kan je gebruiken. Ik zie het als extra regels. Terecht, het is een extra check. Die foreach, daar komt de code niet. Macht de gewoonte. :-)